ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -1506,10 +1529,24 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
             )
         }
 
+        if let cellLeadingView = cellLeadingView {
+            let customViewDimensions: CGFloat = tokenSet[.cellLeadingViewDimensions].float
+            let customViewYOffset = ceil((contentView.frame.height - customViewDimensions) / 2)
+            let customViewXOffset = TableViewCell.customViewLeadingOffset(isInSelectionMode: isInSelectionMode,
```

Comment:
So I totally could have made my own calculation here instead of using the `customViewLeadingOffset` static method here, but I chose this route because of code consistency + cleanliness. I do think that it might have compromised the binary size tho, because now tokens are being called + passed in every where else it's used in this file. Any thoughts would be appreciated :)